<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1615</width>
    <height>983</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMS Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTabWidget" name="component_treewidget">
          <property name="maximumSize">
           <size>
            <width>16777214</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_assets">
           <attribute name="title">
            <string>Assets</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTreeWidget" name="asset_treewidget">
              <property name="contextMenuPolicy">
               <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
              </property>
              <property name="rootIsDecorated">
               <bool>true</bool>
              </property>
              <property name="headerHidden">
               <bool>true</bool>
              </property>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_components">
           <attribute name="title">
            <string>Compoments</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTreeWidget" name="treeWidget_components">
              <property name="headerHidden">
               <bool>true</bool>
              </property>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_configurations">
           <attribute name="title">
            <string>Configuration</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTreeWidget" name="treeWidget_configurations">
              <property name="headerHidden">
               <bool>true</bool>
              </property>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_contacts">
           <attribute name="title">
            <string>Contacts</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTreeWidget" name="treeWidget_contacts">
              <property name="headerHidden">
               <bool>true</bool>
              </property>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QMdiArea" name="mdiArea"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1615</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
